MANIFEST.in

```diff
@@ -3,3 +3,6 @@ include LICENSE.txt
 
 # Include the data files
 recursive-include data *
+
+# Include package data
+include sample/package_data.dat
```

Comment:
Is this just because of the "if you are using Python 2.6, it should be in `MANIFEST.in` comment below? If so, then (a) it's not specific to this PR, it's a change that applies to the existing code as well, but (b) we don't support Python 2.6 according to `setup.py`/`setup.cfg`, so we don't need it anyway.